src/app/layout/footer/footer.component.ts

```diff
@@ -23,7 +23,11 @@ export class FooterComponent implements OnInit {
     })
   }
 
-  ngOnInit() {}
+  ngOnInit() {
+    setTimeout(() => {
+      document.getElementById('ot-sdk-btn').innerText = $localize`:@@footer.cookieSettings:Cookie Settings`
```

Comment:
Dani let's use the window provider to get the document object

Reply:
Also might be a good idea to check if the element exists, before trying to update the `innerText`

Reply:
Done! @leomendoza123